src/components/NcSettingsSection/NcSettingsSection.vue

```diff
@@ -126,22 +115,18 @@ $maxWidth: 900px;
 .settings-section {
 	display: block;
 	margin-bottom: auto;
-	padding: 30px;
+	padding: 0 0 calc(var(--default-grid-baseline) * 5) 0;
+	margin: calc(var(--default-grid-baseline) * 7);
+	width: min($maxWidth, 100%);
```

Comment:
What is `maxWidth` ?

Reply:
I think 900px ...

```suggestion
	width: min(900px, 100%);
```

Reply:
So we no longer allow wider sections? I think there will be complains by app developers as there are even requests for multi column settings (I think because today you have 21:9 and wider displays resulting in a lot of dead space).

Reply:
Ref: https://github.com/nextcloud-libraries/nextcloud-vue/issues/4753

Reply:
I discussed this between designers and we wouldn't want to allow something like that and prefer to contain the column width of the settings section to a max of 900 px.

For particular applications other components or custom wrappers can still be used.